test/TextField.test.ts

```diff
@@ -0,0 +1,67 @@
+import { describe, expect, it } from 'vitest';
+import { render } from '@testing-library/svelte';
+
+import TextField from '../src/lib/components/TextField.svelte';
+
+const INPUT_DEFAULT_CLASSES =
+  'bg-gray-50 border border-gray-300 text-gray-900 sm:text-sm rounded-lg focus:ring-primary-500 focus:border-primary-500 block w-full p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-primary-500 dark:focus:border-primary-500';
+
+describe('Test for TextField component', () => {
+  it('Create input with default props', () => {
+    const { container } = render(TextField);
+    const inputElement = container
+      .getElementsByTagName('input')
+      .item(0) as HTMLElement;
+    const paragraphErrorElement = container
+      .getElementsByTagName('p')
+      .item(0) as HTMLParagraphElement;
+    const labelElement = container
+      .getElementsByTagName('label')
+      .item(0) as HTMLLabelElement;
```

Comment:
Just as a hint! If you want to ensure how its rendered, and preserve certain structure, you can use snapshots.